www/video-rtc.js

```diff
@@ -664,6 +667,23 @@ export class VideoRTC extends HTMLElement {
         this.send({type: 'mp4', value: this.codecs(this.video.canPlayType)});
     }
 
+    static isH265Supported() {
+        try {
+            const videoCodecs = RTCRtpSender?.getCapabilities('video')?.codecs;
+            
+            if (!videoCodecs) {
+            return false;
+            }
```

Comment:
Minor indentation issue which I think can be better fixed this way:

```suggestion
            if (!videoCodecs) return false;
```